pkg/client/unversioned/request_test.go

```diff
@@ -729,6 +734,39 @@ func TestDoRequestNewWay(t *testing.T) {
 	fakeHandler.ValidateRequest(t, requestURL, "POST", &reqBody)
 }
 
+// This test assumes that the client implementation backs off exponentially, for an individual request.
+func TestBackoffLifecycle(t *testing.T) {
+	count := 0
+	testServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		count++
+		t.Logf("attempt %d", count)
+		//requests "1" and "2" both fail.
```

Comment:
Well it makes me think that all other requests will work. I only skimmed over this for now, but it seems that all requests except for 5 and 9 will fail.